index.html

```diff
@@ -1177,119 +1177,269 @@ <h2 id="relationship-to-permissions-policy">
         permission=] or by some user agent policy.
       </p>
     </section>
-    <section data-cite="webdriver">
+    <section>
       <h2 id="automation">
         Automated testing
       </h2>
       <p>
         For the purposes of user-agent automation and application testing, this document defines
-        the following <a>extension commands</a> for the [[WebDriver]] specification. It is OPTIONAL
-        for a user agent to support <a>extension commands</a> commands.
+        extensions to the [[WebDriver]] and [[WebDriver-BiDi]] specifications.
+        It is OPTIONAL for a user agent to support them.
       </p>
       <pre class='idl'>
         dictionary PermissionSetParameters {
           required PermissionDescriptor descriptor;
           required PermissionState state;
         };
       </pre>
-      <section>
-        <h3 id="set-permission-command">
-          Set Permission
+      <p>To <dfn>set a permission</dfn> given <var>descriptor</var>, and <var>state</var>:
```

Comment:
```suggestion
      <p>To <dfn>set a permission</dfn> given |descriptor:PermissionDescriptor|, and |state:PermissionState|:
```

Reply:
Done, thanks! Adding the types looks good